extra-files/convert_junit.py

```diff
@@ -41,17 +41,27 @@ def main():
         tests = 0
         time = 0.0
         for testcase in testcases[classname]:
+            content_element = None
             if testcase.find('error') is not None:
+                content_element = testcase.find('error')
                 testcase.set('status', 'failure')
                 failures += 1
             elif testcase.find('failure') is not None:
+                content_element = testcase.find('failure')
                 testcase.set('status', 'failure')
                 failures += 1
             elif testcase.find('skipped') is not None:
+                content_element = testcase.find('skipped')
                 testcase.set('status', 'skipped')
                 skipped += 1
             else:
                 testcase.set('status', 'success')
+
+            if content_element:
+                timestamp = testcase.get('timestamp')
+                content_element.text = "# timestamp {}\n{}".format(
+                        timestamp, content_element.text)
```

Comment:
This doesn't seem to work. See https://openqa.qubes-os.org/tests/26998. `timestamp` is included in the XML, but not in the textual message.

Reply:
Hum. I thought I had tested this. Should work now. Sorry about that.

Reply:
I'm having troubles to see how the new version is different than the old one (besides moving to a function)? Which part makes it work now, order of operations?